### PR TITLE
Breadcrumbs, edits to Foundations, SCSS cleanup, GH source ref. URL bugfix, etc.

### PR DESCRIPTION
**Pull request by klepas**
This PR holds things. I will figure out what they are momentarily >_<

Documentation edits:

- minor edits to the *Foundations — § Layouts*
- updates palette in *Foundations — § Colours*, and **removes** the functional (prev. ‘contextual’) colour listing — removed due to complexity in representing the information, and the fact that the UI-Kit’s colour partial needs to be updated with correct naming conventions for the colour vars.
- moves commented-out version, date, and the 3 img badges into the footer
- moves the feedback button in the header into the hero (only present on the homepage) (footer feedback unaffected)
- refactors some of the custom SCSS for the *Foundations — § Colours* into a mixin
- other supporting edits to the custom SCSS (`design-guide.scss`).

### CODE REVIEW

**Review by dominikwilkowski — APPROVED** [review]

**Comment by klepas** [comment]
@joolswood Hey, I just wanna get your blessing re. the minor content edits — could you take a quick squiz at the diffs of the MD files? (:

**Comment by klepas** [comment]
Added:

- breadcrumbs! (currently just set to support a tier-2 IA, and usage of Jekyll Collections)
- breadcrumb style: inverted, and set to `$darker-aqua` — think it looks better than `$non-white` (the default) — what do y’all think? :)
- scss source URL bugfix
- changes the beta badge in the header to maroon, as set by the `badge--beta` styling

**Comment by dominikwilkowski** [comment]
The breadcrumbs have a bug where the "overview" page link goes nowhere. Let me see if I can find the code responsible

**Review by dominikwilkowski — CHANGES_REQUESTED** [review]
Breadcrumb bug

File: _includes/header.liquid [hunk 1]
```diff
@@ -25,14 +23,40 @@
...
+        <li><a href="/">Home</a></li>
+        <li><a href="/{{ page.parent }}">{{ page.parent | capitalize }}</a></li>
+        <li>
+          <a href="/{{ page.parent }}/{{ page_title_slugified }}">
+            {% if page_title_slugified == page.parent %}
+              Overview
+            {% else %}
+              {{ page.title }}
+            {% endif %}
+        </a></li>
```
Comment (dominikwilkowski) [inline _includes/header.liquid]: some indentation funkiness?

File: _includes/header.liquid [hunk 2]
```diff
@@ -25,14 +23,40 @@
...
+      for the its page.parent.
+    {% endcomment %}
+    {% capture page_title_slugified %}{{ page.title | slugify }}{% endcapture %}
+    <div class="wrapper">
+      <ul>
+        <li><a href="/">Home</a></li>
+        <li><a href="/{{ page.parent }}">{{ page.parent | capitalize }}</a></li>
+        <li>
+          <a href="/{{ page.parent }}/{{ page_title_slugified }}">
+            {% if page_title_slugified == page.parent %}
```
Comment (dominikwilkowski) [inline _includes/header.liquid]: We could either do:
```liquid
<li>
  {% if page_title_slugified == page.parent %}
    <a href="/{{ page.parent }}/index.html">
      Overview
    </a>
  {% else %}
    <a href="/{{ page.parent }}/{{ page_title_slugified }}">
      {{ page.title }}
    </a>
  {% endif %}
</li>
```

or not link to the last link as that's always going to be the page we are currently on...

```liquid
<li>
  {% if page_title_slugified == page.parent %}
    Overview
  {% else %}
    {{ page.title }}
  {% endif %}
</li>
```

**Review by dominikwilkowski — CHANGES_REQUESTED** [review]
The `overview` bug

**Comment by gazhands** [comment]
Cheers for the update!
Think we could go with default breadcrumb style ;)

File: _includes/header.liquid [hunk 3]
```diff
@@ -40,21 +41,24 @@
       The capture is only done for brevity and future re-use.
 
       The if condition sets the page title to ‘Overview’ if it’s the index page
-      for the its page.parent.
+      for its page.parent.
```
Comment (dominikwilkowski) [inline _includes/header.liquid:45]: nice catch

File: _includes/header.liquid [hunk 4]
```diff
@@ -40,21 +41,24 @@
...
     {% endcomment %}
     {% capture page_title_slugified %}{{ page.title | slugify }}{% endcapture %}
     <div class="wrapper">
       <ul>
         <li><a href="/">Home</a></li>
         <li><a href="/{{ page.parent }}">{{ page.parent | capitalize }}</a></li>
         <li>
-          <a href="/{{ page.parent }}/{{ page_title_slugified }}">
-            {% if page_title_slugified == page.parent %}
+          {% if page_title_slugified == page.parent %}
```
Comment (dominikwilkowski) [inline _includes/header.liquid:53]: So we're going with the link for the last bit, cool

File: _includes/header.liquid [hunk 5]
```diff
@@ -4,7 +4,8 @@
   <section class="page-header">
     <div class="wrapper">
       <div class="page-header__logo">
-        <a href="/" class="logo">{{ site.title }}</a> <span class="badge--beta">beta</span>
+        <a href="/" class="logo">{{ site.title }}
```
Comment (dominikwilkowski) [inline _includes/header.liquid]: I personally make it easy for me to catch those things by indenting all levels unless you only have a string you wrap in a tag.

```html
<a href="/" class="logo">
	{{ site.title }}
	<span class="uikit--version">{% uikit_version %}</span>
</a>
```

But no need to change this... :)

**Review by dominikwilkowski — APPROVED** [review]

**Pull request merged**